docs/install.rst

```diff
@@ -87,28 +88,67 @@ install the dependencies that are available through conda::
 
     %> conda install -c conda-forge numpy scipy matplotlib mpi4py healpy pyephem
 
-Using Configure
+Using CMake
 -----------------------
 
-TOAST uses autotools to configure, build, and install both the compiled code
-and the python tools.  If you are running from a git checkout (instead of a
-distribution tarball), then first do::
+TOAST uses CMake to configure, build, and install both the compiled code
+and the python tools.  The suggested way to run TOAST is to install it in a
+directory within your home folder, and then make it visible using the
+``PYTHONPATH`` environment variable.
```

Comment:
In order to have a functioning install, we also need to add the bin directory to PATH.  I see you added that below, so that is good.